x-pack/plugins/security_solution/public/management/routes.tsx

```diff
@@ -6,17 +6,21 @@
  */
 
 import React from 'react';
+import { QueryClient, QueryClientProvider } from 'react-query';
 import { MANAGEMENT_PATH } from '../../common/constants';
 import { ManagementContainer } from './pages';
 import { SecuritySubPluginRoutes } from '../app/types';
 import { CurrentLicense } from '../common/components/current_license';
+const queryClient = new QueryClient();
```

Comment:
I guess this client should be created somewhere outside and be exportable as it can be useful to create, for example, [persistent queries/mutations](https://react-query.tanstack.com/guides/mutations#persist-mutations) and instead of having a service that makes an API call, we could have persistent named queries/mutations.

Reply:
Yes, at some point. I'd go for doing that the moment we need it instead of trying to support it from the very beginning. I want to have a very simple starting config that I know it works and has very limited complexity.